### PR TITLE
Aerogear 2806

### PR DESCRIPTION
**Pull request by odra**
Adds new sitemap and top navigation items

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -2,58 +2,55 @@
...
         <h5>COMMUNITY</h5>
         <ul class="list-unstyled">
-          <li><a href="https://github.com/aerogear/">GitHub</a></li>
-          <li><a href="/docs/planning/">Jira Repo Issues</a></li>
-          <li><a href="https://lists.jboss.org/mailman/listinfo/aerogear-dev">Developer mailing list</a></li>
-          <li><a href="https://lists.jboss.org/mailman/listinfo/aerogear-users">Users mailing list</a></li>
+          <li><a target="_blank" href="https://github.com/aerogear/">GitHub</a></li>
+          <li><a target="_blank" href="/docs/planning/">Jira Repo Issues</a></li>
+          <li><a  href="/community/contribution-guide/">Contribution Guide</a></li>
+          <li><a target="_blank" href="https://lists.jboss.org/mailman/listinfo/aerogear-dev">Developer mailing list</a></li>
```
Comment (dimitraz) [inline _includes/footer.html]: I think there's just the one google group mailing list now

File: _includes/footer.html [hunk 2]
```diff
@@ -2,58 +2,55 @@
...
-      
       <div class="col-sm-3">
         <h5>COMMUNITY</h5>
         <ul class="list-unstyled">
-          <li><a href="https://github.com/aerogear/">GitHub</a></li>
-          <li><a href="/docs/planning/">Jira Repo Issues</a></li>
-          <li><a href="https://lists.jboss.org/mailman/listinfo/aerogear-dev">Developer mailing list</a></li>
-          <li><a href="https://lists.jboss.org/mailman/listinfo/aerogear-users">Users mailing list</a></li>
+          <li><a target="_blank" href="https://github.com/aerogear/">GitHub</a></li>
+          <li><a target="_blank" href="/docs/planning/">Jira Repo Issues</a></li>
```
Comment (dimitraz) [inline _includes/footer.html]: Is that the right link?

Comment (odra) [reply _includes/footer.html]: I think so - but we will need to update the planning page with the proper links/urls

Comment (dimitraz) [reply _includes/footer.html]: https://issues.jboss.org/browse/AEROGEAR.

Comment (dimitraz) [reply _includes/footer.html]: oh ok cool

**Comment by odra** [comment]
@maleck13 added blank index pages for each section

**Comment by laurafitzgerald** [comment]
@odra is there a link to the sitemap for these changes?

**Comment by odra** [comment]
@laurafitzgerald sitemaps are in this ticket: https://issues.jboss.org/browse/AEROGEAR-2806

**Comment by maleck13** [comment]
chatted directly with @laurafitzgerald. The sitemap is the directory structure.

**Comment by odra** [comment]
So should I roll the html changes back (footer and navigation) and keep the folders/empty files?

**Comment by maleck13** [comment]
I don't think so we would have to change the links anyway. I think there is a different discussion to be had about how these changes make it to the live site. ping @jasonmadigan

**Comment by jasonmadigan** [comment]
@odra @maleck13 yeah I think we can land these as-is

@maleck13 will ping the group offline about how to make changes to the live-site (it's a little convoluted, but I can do it now)

**Comment by laurafitzgerald** [comment]
@jasonmadigan @odra there's some [discussion](https://groups.google.com/forum/#!topic/aerogear/i03kuN3ocC8) about how we can use the branches. Having a seperate feature branch might not be the best solution - see the linked thread for details

**Comment by jasonmadigan** [comment]
@laurafitzgerald missed that one, thanks, OK lets merge to `master`. I think this may change again when we re-visit where this is hosted (I'd personally prefer we use GH pages like feedhenry.org again for ease of access & pushing).

File: _includes/footer.html [hunk 3]
```diff
@@ -2,58 +2,55 @@
...
       <div class="col-sm-3">
         <h5>COMMUNITY</h5>
         <ul class="list-unstyled">
-          <li><a href="https://github.com/aerogear/">GitHub</a></li>
-          <li><a href="/docs/planning/">Jira Repo Issues</a></li>
-          <li><a href="https://lists.jboss.org/mailman/listinfo/aerogear-dev">Developer mailing list</a></li>
-          <li><a href="https://lists.jboss.org/mailman/listinfo/aerogear-users">Users mailing list</a></li>
+          <li><a target="_blank" href="https://github.com/aerogear/">GitHub - Aerogear</a></li>
+          <li><a target="_blank" href="https://github.com/aerogearcatalog/">GitHub - Aerogear Catalog</a></li>
+          <li><a target="_blank" href="/docs/planning/">Jira Repo Issues</a></li>
```
Comment (laurafitzgerald) [inline _includes/footer.html]: @odra could this one link out to the AEROGEAR Jira Project. Right now this links to a page which is out of date.

File: _includes/footer.html [hunk 4]
```diff
@@ -2,58 +2,55 @@
...
+          <li><a  href="/community/contribution-guide/">Contribution Guide</a></li>
```
Comment (laurafitzgerald) [inline _includes/footer.html]: @odra this link is broken now. Will there be a new page included for this?

File: _includes/footer.html [hunk 5]
```diff
@@ -2,58 +2,55 @@
...
         <h5>FOLLOW US</h5>
         <ul class="list-inline">
-          <li><a href="https://www.twitter.com/aerogears"><i class="fa fa-twitter-square"></i> Twitter</a></li>
-          <li><a href="https://www.facebook.com/AeroGearProject"><i class="fa fa-facebook-square"></i> Facebook</a></li>
-          <li><a href="https://plus.google.com/106588661437153585722/posts"><i class="fa fa-google-plus-square"></i> Google+</a></li>
-          <li><a href="/feed.atom"><i class="fa fa-rss-square"></i> Feed</a></li>
+          <li><a target="_blank" href="https://www.twitter.com/aerogears"><i class="fa fa-twitter-square"></i> Twitter</a></li>
+          <li><a target="_blank" href="https://www.facebook.com/AeroGearProject"><i class="fa fa-facebook-square"></i> Facebook</a></li>
+          <li><a target="_blank" href="https://plus.google.com/106588661437153585722/posts"><i class="fa fa-google-plus-square"></i> Google+</a></li>
+          <li><a target="_blank" href="/feed.atom"><i class="fa fa-rss-square"></i> Feed</a></li>
```
Comment (laurafitzgerald) [inline _includes/footer.html]: This one seems to become invisible from time to time and then shows when it's hovered over.

File: _includes/footer.html (hunk 4 above)
Comment (odra) [reply _includes/footer.html]: oops, missed that one, gonna add it

File: _includes/footer.html (hunk 5 above)
Comment (odra) [reply _includes/footer.html]: it it's a general css issue for all links located at the bottom of the page - will file a ticket for that

**Comment by odra** [comment]
@laurafitzgerald I've applied the requested changes and changed the branch to used to master.

**Comment by laurafitzgerald** [comment]
@odra thanks for the changes. I was able to view the pages by manually navigating and they look fine. The link's are not yet updated in the top navigation but I think this is fine as we need to populate the pages first. Or maybe it makes sense to update them now along with the other ones?

**Comment by laurafitzgerald** [comment]
I've gone ahead and approved the changes if this one needs to be merged to progress other work.

**Pull request merged**